packages/backend/src/handlers/triggers/syncDiscordGuildMembers.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable no-console */
+import {
+  createDiscordClient,
+  GuildDiscordMetadata,
+} from '@metafam/discord-bot';
+
+import {
+  Guild,
+  Guild_Player_Insert_Input,
+  GuildStatus_Enum,
+  SyncGuildMembersMutation,
+} from '../../lib/autogen/hasura-sdk';
+import { client } from '../../lib/hasuraClient';
+import { TriggerPayload } from './types';
+
+export const syncDiscordGuildMembers = async (
+  payload: TriggerPayload<Guild>,
+) => {
+  const { new: guild } = payload.event.data;
+
+  if (guild?.discord_id == null) return;
+
+  console.log(`updating guild members for ${guild?.name} from Discord...`);
```

Comment:
```suggestion
  console.debug(`updating guild members for ${guild?.name} from Discord...`);
```

Reply:
This trigger shouldn't run that often and has major effects, otherwise I wouldn't be logging at all